src/coreclr/jit/codegenarm64.cpp

```diff
@@ -6362,6 +6362,31 @@ void CodeGen::genArm64EmitterUnitTests()
     theEmitter->emitIns_R_R(INS_rev, EA_4BYTE, REG_R10, REG_R5);
     theEmitter->emitIns_R_R(INS_rev16, EA_4BYTE, REG_R11, REG_R6);
 
+    // TODO-SVE: Fix once we add predicate registers
+    // decp
+    theEmitter->emitIns_R_R(INS_sve_decp, EA_8BYTE, REG_R0, REG_R1, INS_OPTS_8B); // DECP    <Xdn>, <Pm>.<T>
+    theEmitter->emitIns_R_R(INS_sve_decp, EA_8BYTE, REG_R0, REG_R1, INS_OPTS_8B); // DECP    <Zdn>.<T>, <Pm>.<T>
```

Comment:
If this takes a `Z` register than it should be `REG_V0` instead of `REG_R0`?

Reply:
Yes, thanks for catching that.

Reply:
Done. Please use the new files in https://github.com/dotnet/runtime/issues/94549